Resources/doc/reference/troubleshooting.rst

```diff
@@ -106,3 +106,27 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
             local:
                 directory:  "%kernel.root_dir%/../web/uploads/media"
                 create:     false
+
+ 
+Error on cache:warmup
+---------------------
+
+You may encounter the following error when warming up the cache if you don't have the admin bundle installed:
+
+.. code-block:: text
+
+    Fatal error: Class 'Sonata\AdminBundle\Controller\CRUDController'
+    not found in .....\vendor\sonata-project\media-bundle\Controller\GalleryAdminController.php on line 19
+    
+This problem is most likely caused by the JMSDiExtraBundle's cache warmer (https://github.com/schmittjoh/JMSDiExtraBundle/blob/master/HttpKernel/ControllerInjectorsWarmer.php).
```

Comment:
Either `JMSDiExtraBundle's cache warmer` or `the JMSDiExtraBundle cache warmer`, but not both (I prefer the former). Also, try to keep lines under 120 characters.